test/parallel/test-repl-definecommand.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+require('../common');
+
+var stream = require('stream'),
+    assert = require('assert'),
+    repl = require('repl');
+
```

Comment:
Sure no prob. I saw there was a lint error in CI so I'll fix that and this.

Should I `const` everything in this file that can be? Or just the `require`s?